### ISSUE TITLE
Text gets cut off when the AccessibilityRenderExtension is used and the font scale is set to 2f

### ISSUE THREAD

**Issue by palpapp04** [cashapp/paparazzi#1601]
**Description**
I have a snapshot test where I am using the AccessibilityRenderExtension and I have a font scale test parameter to create multiple snapshots with different font sizes. When the font scale is set to 2f and the AccessibilityRenderExtension is used texts on buttons get cut off. However when I do not use the AccessibilityRenderExtension the text on the buttons does not get cut off.

**Steps to Reproduce**
Here is a sample project I created where the issue is reproducible by running the recordPaparazzi task.
It contains 2 test classes one with AccessibilityRenderExtension and one without.
When you run recordPaparazzi it will generate the snapshots (although the ones I generated are also contained in the zip file if I am correct) and you can see that the snapshots look the same except when the font scale is set to 2f.
[PaparazziTest.zip](https://github.com/user-attachments/files/17114072/PaparazziTest.zip)

**Expected behavior**
My expectation would be that the generated snapshots are the same with and without the AccessibilityRenderExtension and that the text on the buttons does not get cut off when you use the AccessibilityRenderExtension

**Additional information:**
- Paparazzi Version: 1.3.4
- OS: Mac OS Sonoma 14.5
- Compile SDK:  34
- Gradle Version: 8.8
- Android Gradle Plugin Version: 8.5.2

**Screenshots**
![com example paparazzitest_ExampleUnitTest_Example test where text is cut off MAX_FONT_CONFIG](https://github.com/user-attachments/assets/f0f53860-14d8-484f-be2a-0886f50252cf)
![com example paparazzitest_ExampleWithoutA11yExtensionUnitTest_Example test where text is not cut off MAX_FONT_CONFIG](https://github.com/user-attachments/assets/e8a873b1-9e1c-47a8-ae7e-262f086b03ff)

**Comment by palpapp04** [comment]
Thanks in advance to anyone for helping to resolve this issue :)

**Comment by geoff-powell** [comment]
Yep, so the issue is `animationContentSize` modifier `.animateContentSize(animationSpec = tween(durationMillis = 500))` is animating so the initial snapshot is the first frame of animation. Try using `paparazzi.gif()` to record the animation or allow a param to omit animation. We are working on creating an `InstantAnimationRule` that support compose animations soon hopefully 🙏🏽 https://github.com/cashapp/paparazzi/issues/1261 .

**Comment by palpapp04** [comment]
Thanks for your response. Then I guess I will just have to wait for compose animation support. But it is interesting that the issue only comes forward when the AccessiblityRenderExtension is added. Is there some reason for that? Because then I can just not use that extension while there is no support for compose animations but I am a little worried that this issue will come forward here and there without the extension as well.
